addon/components/paper-text-field.js

```diff
@@ -0,0 +1,76 @@
+import Component        from 'ember-component';
+import formFieldOptions from
+  'ember-railio-form-components/mixins/form-field-options';
+import get   from 'ember-metal/get';
+import set   from 'ember-metal/set';
+
+export default Component.extend(formFieldOptions, {
+
+  focusIn() {
+    if (typeof this.attrs.focusIn === 'function') {
+      this.attrs.focusIn();
+    }
+  },
+
+  focusOut() {
+    let _value = get(this, '_value');
+    if (typeof this.attrs.focusOut === 'function') {
+      this.attrs.focusOut();
+    }
+
+    if (typeof get(this, 'format') === 'function') {
+      _value = this.format(_value);
+      set(this, '_value', _value);
+    }
+
+    this.send('changed', _value);
+  },
+
+  keyUp(e) {
```

Comment:
Why not just bind these functions to the component (just like we discussed earlier).

Reply:
I think we discussed this, while you were sitting next to me